README.md

```diff
@@ -1 +1,89 @@
-# ovirt-ansible-engine-setup
+oVirt Engine Setup
+==================
+
+Install required packages for oVirt Engine deployment, generates answerfile
+and runs engine-setup with it.
+Optionally the role can update oVirt engine packages.
+
+Target Systems
+--------------
+
+* engine
+
+Requirements
+------------
+
+ * Environment with configured repositories.
+ * Ansible version 2.3
+
+Role Variables
+--------------
+
+By default engine-setup uses answer file specific for version of oVirt,
+based on ``ovirt_engine_setup_version`` parameter. You can specify own answer file
+as ``ovirt_engine_setup_answer_file_path``.
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_version            | UNDEF                 | Allowed versions: [3.6, 4.0, 4.1, 4.2] |
```

Comment:
When we support all of those version, we should probably document which option below is valid for which version. But it can be done later in subsequent patch

Reply:
Sounds good.